canonical-web-and-design/jaas-dashboard
# Align tables columns in the machine details

## Done
Added bespoke styling to the tables in the machine details as maas-ui does. 

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to the model details
- Check the tables match the tables visible in [the design](https://app.zeplin.io/project/5d2dfd8e5b67f14c5e1d2aef/screen/5e67bbca397a5b129eb35110)

## Details
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/424
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/426

## Screenshots
![Screenshot_2020-05-28 JAAS Dashboard](https://user-images.githubusercontent.com/1413534/83168893-1c042500-a10a-11ea-88e3-c6cd8288bb65.png)
